### ISSUE TITLE
Move secure connection configuration to a CLI option.

### ISSUE THREAD

**Issue by scofalik** [benderjs/benderjs#236]
In my opinion it feels akward to use configuration file to decide whether Bender should start HTTPS server instead of HTTP. SSL/HTTPS feels more like a "mode", not a config option. When I want to start HTTPS bender server I feel like I want to run something like `bender server run -H 0.0.0.0 -p 1030 --secure`. Since we already pass host and port, `secure` flag feels like a natural extension. It is also easier to use as you don't need to change config file.

I propose to remove `secure` flag from configuration and add `-s` / `--secure` option.

**Comment by Reinmar** [comment]
IMO most of the time this isn't a mode. This is just a configuration. Imagine you have a project which you always want to test in this mode, simply because it normally works in this environment. Then, being forced to pass this flag every single time you start Bender would be irritating. The same actually goes to `-H 0.0.0.0`. Both of these options needs to be configurable (and host could default to `0.0.0.0`).

**Comment by scofalik** [comment]
Do you think it is a good idea to have both configurable option and a flag to override the config option in CLI?

**Comment by Reinmar** [comment]
I think it's a question related to https://github.com/benderjs/benderjs/issues/189 and https://github.com/benderjs/benderjs/issues/217.

**Comment by pjasiun** [comment]
I agree that `-s` should works the same way `-H` or even `-p` works. The port is a good example: we use 10vvv port for build version (e.g. 10451 for 4.5.1 version) if the port would be a configuration option builder could add it to the config file (it modify the `bender.js` config anyway) so simple `bender run` will run server with the proper parameters.

**Comment by pjasiun** [comment]
So +1 for having `-s` / `--secure`, the API should be consistent. But also all options should have defaults in the config file.
